pennylane_lightning/src/util/Kokkos_Sparse.hpp

```diff
@@ -47,7 +47,7 @@ using const_index_view_type =
     typename Kokkos::View<const index_type *, default_layout, device_type,
                           Kokkos::MemoryTraits<Kokkos::Unmanaged>>;
 
-template <class fp_precision> using data_type = Kokkos::complex<fp_precision>;
+template <class fp_precision> using data_type = std::complex<fp_precision>;
```

Comment:
don't we need to use `Kokkos::complex` anymore?

Reply:
Since we only do this for CPUs I think we don't need the `Kokkos::complex` here anymore. My understanding is that matters more if we intend on compiling for GPUs too --- though correct me if wrong @AmintorDusko

Reply:
Hi @chaeyeunpark. It is as @mlxd said. For now, std::Complex serve us well. It removes the necessity of typecasting and reduces memory usage.